docs/_articles/en/codebuilder/cb-setup.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Set Up Code Builder
+lang: en
+---
+
+## Install the Code Builder Managed Package (Ss much detail is not necessary?)
+
+Salesforce Code Builder is a second-generation (2GP) managed package that you can install in any supported Salesforce org edition. Before you install, make sure you have enabled **Unlocked Packages and Second-Generation Managed Packages** in your Dev Hub.
+
+1. Click the install [link](https://login.salesforce.com/packaging/installPackage.apexp?p0=04t6g000008nwJuAAI)
+2. Log into the org in which you want to install Code Builder.
+3. Check Acknowledge and click **Install**.
+
+
+
+
+![Install Button](../../../images/install_button.png)
+
+4. Grant access and click **Continue**
+
+
+
+![Grant Access](../../../images/grant_access.png)
+
+
+
+5. Click **Done** when installation is Complete.
+
+## Install Code Builder Managed Package (just use this?)
+
+Salesforce Code Builder is a second-generation (2GP) managed package that you can install in any supported Salesforce org edition. Before you install, make sure you have enabled **Unlocked Packages and Second-Generation Managed Packages** in your Dev Hub then follow the standard process for installing a managed package. 
```

Comment:
Assuming for beta, at least, that if you have to install it in a dev hub, then it's the 4 supported dev hub editions.